osquery/core/watcher.cpp

```diff
@@ -128,8 +138,25 @@ CLI_FLAG(uint64,
 
 CLI_FLAG(bool, disable_watchdog, false, "Disable userland watchdog process");
 
+CLI_FLAG(bool,
+         enable_watchdog_debug,
```

Comment:
I'd remove `enable`, since I think it's always confusing. But I don't feel strongly about it